prime-router/metadata/hl7_mapping/ORU_R01/base/observation-result.yml

```diff
@@ -18,6 +18,12 @@ elements:
     value: [ '"CE"' ]
     hl7Spec: [ '%{hl7OBXField}-2' ]
 
+  - name: result-value-type-cwe
+    condition: '%resource.identifier.type.coding.code.exists() and %resource.identifier.type.coding.code = "81959-9"'
```

Comment:
I actually don't think this is going to work to add the CWE instead. It looks like there are many LOINC codes for AOE questions. https://loinc.org/sars-cov-2-and-covid-19/ I'm wondering if we actually need to make the AOE questions be CWEs instead of just CEs

Reply:
I don't think you need to look at LOINC codes. You should only look at the type of result and derive this field from that

Reply:
Removed this change, AOE questions are type CE now.